features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapNotificationImpl.java

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Delete this class, you don't need to create a new TrapNotification implementation. Please talk to Pradeep about how to create Snmp4JV1TrapInformation or Snmp4JV2TrapInformation objects for testing (which implement TrapNotification).